test/e2e/azureidentity/azureidentity.go

```diff
@@ -0,0 +1,190 @@
+package azureidentity
+
+import (
+	"context"
+	"encoding/json"
+	"html/template"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"time"
+
+	aadpodid "github.com/Azure/aad-pod-identity/pkg/apis/aadpodidentity/v1"
+	"github.com/Azure/aad-pod-identity/test/e2e/util"
+	"github.com/pkg/errors"
+)
+
+// CreateOnAzure will create a user-assigned identity on Azure, assign 'Reader' role
```

Comment:
Can you please add a comment at the beginning of each file for what kind of test it covers, what are the bugs we expect to catch etc.